src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -53,20 +56,20 @@
          [quo/account-overview
           {:current-value       (utils/prettify-balance balance)
            :account-name        name
-           :account             :default
+           :account             (if watch-only? :watched-address :default)
            :customization-color color}]
          [quo/wallet-graph {:time-frame :empty}]
-         [quo/wallet-ctas
-          {:send-action   #(rf/dispatch [:open-modal :wallet-select-address])
-           :buy-action    #(rf/dispatch [:show-bottom-sheet
-                                         {:content buy-drawer}])
-           :bridge-action #(rf/dispatch [:open-modal :wallet-bridge])}]
+         (when (not watch-only?)
+           [quo/wallet-ctas
+            {:send-action   #(rf/dispatch [:open-modal :wallet-select-address])
+             :buy-action    #(rf/dispatch [:show-bottom-sheet
+                                           {:content buy-drawer}])
+             :bridge-action #(rf/dispatch [:open-modal :wallet-bridge])}])
          [quo/tabs
-          {:style            style/tabs
-           :size             32
-           :default-active   @selected-tab
-           :data             tabs-data
-           :on-change        #(reset! selected-tab %)
-           :scrollable?      true
-           :scroll-on-press? true}]
```

Comment:
Why remove `scroll-on-press?` prop?

Reply:
@mmilad75 made this change,. do you recall if it's required? 🤔

Reply:
Yes it is required. This is to keep the selected tab centered. (similar to tabs in AC)

Reply:
thanks @OmarBasem. will put it back! 🙏

Reply:
restored 👍